lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py

```diff
@@ -141,6 +142,10 @@ def __init__(self):
                 type='int',
                 required=False
             ),
+            managed_by=dict(
```

Comment:
Do we need this extra option in the facts module?

Reply:
Good point, I will add it

Reply:
It is already there

Reply:
I think so, since this is a field returned from the API, and also now is added to the module parameters

Reply:
Ok, we would have to add it to the documentation part as well.